source/sass/components/article-page.scss

```diff
@@ -204,8 +204,8 @@
     margin-top: 47px;
   }
 
-  hr ~ img {
-    margin-top: 0 !important;
+  hr ~ img.full-width {
+    margin-top: 0;
```

Comment:
only `img.full-width` comes with top margin so we only have to target that selector instead of all `img`

Reply:
test article page: https://foundation-s-issue-6355-xnjp28.herokuapp.com/en/publication-page-with-child-article-pages/fixed-title-article-page/

Inspect and verify that the 3 turquoise images don't have top margin.

<img width="301" alt="image" src="https://user-images.githubusercontent.com/2896608/112394773-7e8e6900-8cba-11eb-9087-d4305ce6e983.png">